hknweb/templates/about/abouthkn.html

```diff
@@ -0,0 +1,91 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block header %}
+<link rel="stylesheet" href="{% static 'css/style.css' %}">
+
+{% endblock %}
+
+{% block title %}About HKN{% endblock %}
+
+
+{% block heading %}
+About HKN
+{% endblock %}
+
+{% block content %}
+<div class='centered'>
+<body>
+  <div class='row'>
+
+    <div class='column one'>
+        <p class='what-about-subtitle'>Eta Kappa Nu (HKN) is the <a href='https://hkn.ieee.org/' target='_blank'>national Electrical and Computer Engineering honor society</a>. Founded in 1915, the Mu chapter at UC Berkeley is among the most active engineering societies at Berkeley. We strive to serve the Electrical Engineering and Computer Science (EECS) community at Berkeley by providing weekly tutoring hours, exam review sessions, department tours, course and staff evaluations, among many other services. Our two offices are located in 290 Cory and 345 Soda near the <a href='https://berkeley.edu/map' target='_blank'>northeastern corner of the UC Berkeley campus</a>.</p>
+      </div>
+      <div class='column two'>
+        <a href="https://hkn.ieee.org/get-involved/awards/outstanding-chapter-award/" target="_blank" align='center'><p>
+          <img id="outstanding-chapter" src="{% static 'img/outstanding_chapter.png' %}" alt="2019 Outstanding Chapter Award" width=100%></p>
+          </a>
+      </div>
+      
+  </div>
+
+  <div class="tabset">
+  <!-- Tab 1 -->
+  <input type="radio" name="tabset" id="tab1" aria-controls="services" checked>
+  <label for="tab1">Student Services</label>
+  <!-- Tab 2 -->
+  <input type="radio" name="tabset" id="tab2" aria-controls="indrel">
+  <label for="tab2">Industrial Relations</label>
+  <!-- Tab 3 -->
+  <input type="radio" name="tabset" id="tab3" aria-controls="community">
+  <label for="tab3">Community</label>
+  
+  <div class="tab-panels">
+    <section id="services" class="tab-panel">
+      <div class='tabpadding'>
+        <div class='row'>
+        <div class='column half'>
```

Comment:
Indentation is also a bit strange here (and cascades into some sections below)

Not a big deal though